org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui.tests/src/org/eclipse/xtext/example/domainmodel/ui/tests/ContentAssistTest.xtend

```diff
@@ -56,7 +52,7 @@ class ContentAssistTest extends AbstractContentAssistTest {
 			entity E {
 				«c»
 			}
-		'''.testContentAssistant(#[
+		'''.assertContentAssistant(#[
```

Comment:
OK, so, concerning my comment above about `String[]` instead of `List`, this seems to work out of the box with Xtend, right?

Reply:
Yes, Xtend automatically converts the actual parameter from array to list (and vice versa) if the formal parameter is a list/array.